website/pages/docs/job-specification/upstreams.mdx

```diff
@@ -80,6 +81,8 @@ job "countdash" {
 - `destination_name` `(string: <required>)` - Name of the upstream service.
 - `local_bind_port` - `(int: <required>)` - The port the proxy will receive
   connections for the upstream on.
+- `datacenter` `(string: <optional>)` - The datacenter in which to issue the
+  discovery query. Defaults to the local datacenter.
```

Comment:
Can we be more specific about "local" here? The default looks like it ends up as `""` (empty string). Is that the default datacenter for the server's Consul agent, or...?